client/src/lib/utils/formats.ts

```diff
@@ -1,16 +1,27 @@
 export function formatPercentage(value: number, options?: Intl.NumberFormatOptions) {
-  const v = Intl.NumberFormat('en-US', {
+  const formatter = Intl.NumberFormat('en-IE', {
     minimumFractionDigits: 0,
     maximumFractionDigits: 2,
     style: 'percent',
     ...options,
   });
 
-  return v.format(value);
+  return formatter.format(value);
+}
+
+export function formatDate(value: string, options?: Intl.DateTimeFormatOptions) {
```

Comment:
Since you're working on this already, Cirad would like the following format for the dates: «01/Jan/2023».

There is a [separate task](https://vizzuality.atlassian.net/browse/ORC-184) for the next sprint, but if that's not hard to include, that'd be nice.

Reply:
Feel free to ignore this comment for now.